partcad-cli/src/partcad_cli/click/commands/status.py

```
@@ -8,17 +8,27 @@
 
 
 import rich_click as click
+import sentry_sdk
 
 import os
 import threading
+import socket
```

Ruff (0.8.2)

`socket` imported but unused

Remove unused import: `socket`

(F401)